recipes-core/initrdscripts/initramfs-framework/bootfs

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+# Copyright (C) 2018 Apertus Solutions, LLC
+# Licensed on MIT
+
+bootfs_enabled() {
+	if [ -n "${bootparam_boot}" ]; then
+		return 0
+	else
+		return 1
+	fi
+}
+
+bootfs_run() {
+	BOOTFS_DIR="/${ROOTFS_DIR}/boot/system"
+        if [ -z "$BOOTFS_DIR" ]; then
+                return
+        fi
+        C=0
+	# will use rootfs' delay and timeout values
```

Comment:
Do we require bootfs to always be on the same device as rootfs?  If we require the same device, then the delay is unneeded since rootfs would already handle it.  However, I can understand leaving it to track the upstream rootfs script.

Also, here and throughout the PR, there is a mix of spaces and tabs.

Reply:
@jandryuk The mix of spaces and tabs is coming from the upstream script.

Reply:
I left the bootfs delay as it is in grub.cfg today. Do we want it to go away?